hooks/useSocketAssets.ts

```diff
@@ -15,46 +16,63 @@ const ETH = {
   icon: '/img/assets/WETH.svg',
   logoURI: '/img/assets/WETH.svg',
   amount: 0,
+  usdAmount: 0,
 } satisfies AssetBalance;
 
+const VELO = {
+  chainId: 10,
+  address: '0x9560e827aF36c94D2Ac33a39bCE1Fe78631088Db',
+  name: 'Velodrome V2',
```

Comment:
only _Velodrome_

Reply:
the price of it is also not working, and we have that price (from socket)

Reply:
ok, I can fetch it